content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -0,0 +1,460 @@
+---
+title: Server-Side Apply
+reviewers:
+- smarterclayton
```

Comment:
```suggestion
- smarterclayton
- apelisse
```

Reply:
Add "apelisse"

Reply:
good point.

Reply:
And this page has its own item on the navigation bar.